From 59859acf9083010639eb9bc0bf8cb5d03f03935f Mon Sep 17 00:00:00 2001
From: SiYao Mo <msylgj@vip.qq.com>
Date: Fri, 16 Jul 2021 14:25:44 +0800
Subject: [PATCH] pdnsd-alt: fix cmake error in kernel 5.13/5.14 Because of
 redefinition of 'struct ifmap' & 'struct ifreq' & 'struct ifconf' in
 linux/if.h (Linux Kernel above 5.13) Replace net/if.h with linux/if.h

Signed-off-by: SiYao Mo <msylgj@vip.qq.com>
---
 src/conff.h | 2 +-
 src/dns.h   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/conff.h b/src/conff.h
index a07b156..c66d210 100644
--- a/src/conff.h
+++ b/src/conff.h
@@ -32,7 +32,7 @@
 #include <stdio.h>
 #include <pthread.h>
 #include <sys/socket.h>
-#include <net/if.h>
+#include <linux/if.h>
 #include "ipvers.h"
 #include "list.h"
 
diff --git a/src/dns.h b/src/dns.h
index 0f6a4ac..ecc9680 100644
--- a/src/dns.h
+++ b/src/dns.h
@@ -27,7 +27,7 @@
 #include <config.h>
 #include <arpa/inet.h>
 #include <sys/socket.h>
-#include <net/if.h>
+#include <linux/if.h>
 #include <sys/types.h>
 #include <inttypes.h>
 #include "rr_types.h"
